en/02_Developer_Guides/08_Performance/01_Caching.md

```diff
@@ -255,19 +256,16 @@ with the `disable-container` argument.
 ```yaml
 SilverStripe\Core\Injector\Injector:
   Psr\SimpleCache\CacheInterface.myapp:
-    factory: SilverStripe\Core\Cache\CacheFactory     
+    factory: SilverStripe\Core\Cache\CacheFactory
     constructor:
       namespace: "MyInsensitiveData"
       disable-container: true 
-``` 
+```
 
 ## Additional Caches
 
 Unfortunately not all caches are configurable via cache adapters.
 
  * [SSViewer](api:SilverStripe\View\SSViewer) writes compiled templates as PHP files to the filesystem
    (in order to achieve opcode caching on `include()` calls)
- * [ConfigManifest](api:SilverStripe\Core\Manifest\ConfigManifest) is hardcoded to use `FilesystemCache`
- * [ClassManifest](api:SilverStripe\Core\Manifest\ClassManifest) and [ThemeManifest](api:SilverStripe\View\ThemeManifest)
```

Comment:
Have removed references to Config + Class manifests because they're not hardcoded to use a particular type of cache